fiftyone/core/odm/database.py

```diff
@@ -1839,6 +1840,12 @@ def _get_fcv_and_version(
         return current_fcv, server_version
     except ServerSelectionTimeoutError as e:
         raise ConnectionError("Could not connect to `mongod`") from e
+    except:
```

Comment:
Are we sure we want to catch every error to respond with this message?

Reply:
It's definitely heavy-handed. But the actual error raised here could either be from `pymongo` itself (OSS) or from the pymongo proxy (Teams). IDK how the latter works.

Reply:
Currently the proxy will return a no access error for any database that isn’t the fiftyone database

Reply:
I say, let's fix in a follow-up, but let's get this patch release out today.

Reply:
Net-net, we're saying that FCV version checking is an optional service we provide. So I like the generic except here. If some other exception occurred, I wouldn't want to disrupt users either.